includes/validation/class-amp-validation-callback-wrapper.php

```diff
@@ -323,7 +323,7 @@ protected function finalize_scripts( WP_Scripts $wp_scripts, array $before_regis
 			$handles = [ $added_handle ];
 
 			// Account for case where registered script is a placeholder for a set of scripts (e.g. jquery).
-			if ( isset( $wp_scripts->registered[ $added_handle ] ) && false === $wp_scripts->registered[ $added_handle ]->src ) {
+			if ( isset( $wp_scripts->registered[ $added_handle ] ) && empty( $wp_scripts->registered[ $added_handle ]->src ) ) {
```

Comment:
Below message, I'm getting while running the phpstan.
```
------ ------------------------------------------------------------------------------------- 
  Line   includes/validation/class-amp-validation-callback-wrapper.php                        
 ------ ------------------------------------------------------------------------------------- 
  326    Result of && is always false.                                                        
  326    Strict comparison using === between false and string will always evaluate to false.  
 ------ ------------------------------------------------------------------------------------- 
 ```

Yes, As per documentation in core `_WP_Dependency::$src` have only a string value.

Reply:
Seems like that's something that WordPress core should fix in its phpdoc as well, but I don't see any downside to using `empty()` instead of checking for `false`.